docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -0,0 +1,60 @@
+# Troubleshooting Common Installation Errors
+
+This guide assumes installation on any supported OS.  
+
+This guide offers support for common issues faced when first installing LORIS. It is assumed that all installation steps provided have been 
+correctly followed and that all required dependencies have been installed.
+
+## Frontend Does Not Load/JavaScript changes not visible
+The first thing to try is to run:
+
+```bash
```

Comment:
Before the `make`/`make dev` step could you add `make clean`? This sometimes fixes problems the other commands miss, for example this was a solution when #6220 was an issue.